index.js

```diff
@@ -2195,6 +2196,36 @@
         return filter(is(type), Just(x));
       });
 
+  //. ### Numeric
+
+  //# add :: FiniteNumber -> FiniteNumber -> FiniteNumber
+  //.
+  //. Adds two finite numbers together.
+  //.
+  //. ```javascript
+  //. > S.add(1, 1);
+  //. 2
+  //. ```
+  S.add =
+  def('add',
+      {},
+      [$.FiniteNumber, $.FiniteNumber, $.FiniteNumber],
+      function(a, b) { return a + b; });
+
+  //# subtract :: FiniteNumber -> FiniteNumber -> FiniteNumber
```

Comment:
`s/subtract/sub/`

Could you update the commit message and pull request title to match? :)